instance creation
new
	self error: 'Invalid invocation'